src/state/selectors.ts

```diff
@@ -61,7 +61,7 @@ export const selectParamFromFilter = <T extends ParamFilterKey>(param: T) =>
     param => param,
   )(
     (_state: ReduxState, filter: Pick<ParamFilter, T> | null) =>
-      `${param}-${filter?.[param]}` ?? param,
+      `${param}-${filter?.[param] ?? param}`,
```

Comment:
if `filter` is undefined we get `'param-undefined'`. Coalescing did nothing here. But it's a valid string unique to the failure case so was not actually broken. The change here simply aligns the implementation with the intention.

Reply:
feels like intention would be:
```
filter ? `${param}-${filter.[param]}` : param
```
so it is either `param` or `param-filter`?

Reply:
I guess yours does match more closely the style in the selectors below now that I am looking at them